src/status_im2/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -0,0 +1,136 @@
+(ns status-im2.contexts.wallet.send.select-asset.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.send.select-asset.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def tabs-data
+  [{:id :tab/assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
+   {:id :tab/collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}])
+
+(defn- network-names
```

Comment:
can we remove this mock data, we already have this with`:wallet/network-details`. 
Not sure where balances per-chain comes in but I believe it should be it's own sub in that case too 👍 which perhaps we already have / is in progress in another pr?

Reply:
Sure, will explore how to handle this better with subs